app/src/main/res/values/strings.xml

```diff
@@ -280,22 +280,22 @@
     <string name="settings_title_notification_coloration">Color notification icon</string>
     <string name="file_is_empty">File is empty</string>
     <string name="install_with_su">Silent installation (root)</string>
-    <string name="install_with_su_summ">If checked EdXposed Manager will install modules without a prompt</string>
+    <string name="install_with_su_summ">If checked, EdXposed Manager will install modules without any prompt</string>
     <string name="install_remove">Install and remove</string>
     <string name="download_remove">Download and remove</string>
     <string name="module_saved">Module saved at %1$s</string>
     <string name="ignore_updates">Ignore update</string>
     <string name="ignore_updates_summ">Ignore updates for a specific module</string>
     <string name="reboot_recovery">Reboot to Recovery</string>
     <string name="reboot_bootloader">Reboot to Bootloader</string>
-    <string name="reboot_download">Reboot to Download</string>
-    <string name="reboot_edl">Reboot to Emergency</string>
+    <string name="reboot_download">Reboot to Download Mode</string>
+    <string name="reboot_edl">Reboot to EDL</string>
```

Comment:
This mode has many aliases, such as EDL and 9008, and E is Emergency, so I prefer to use Emergency or Emergency Mode